src/client.js

```diff
@@ -476,6 +476,119 @@ class ConnectionParameters {
 
 }
 
+// ========================================================================================
+// File Uploader
+// ========================================================================================
+
+const fileUploader = (client) => {
+  /**
+   * Will increament the counter
+   * @returns {Promise<number|*>}
+   */
+  const increaseValue = async () => {
+    const soapCall = client._prepareSoapCall('xtk:counter', 'IncreaseValue')
```

Comment:
Other APIs in **client.js** are also using **_prepareSoapCall** for making soap calls. Is there any reason to use xtkSession instead of _prepareSoapCall?